tox.ini

```diff
@@ -9,30 +9,23 @@ envlist = py27,lint
 [base]
 deps =
     ipaddress
-    salt
+    salt<2017.0.0
 
 [testenv:py27]
-basepython = python2.7
-commands =
-    py.test --cov=. --tb=line -v --ignore=cli/ --junitxml=junit-{envname}.xml {posargs}
-    codecov
-# the environment variables are needed by codecov
-passenv = CI TRAVIS TRAVIS_*
+commands =  py.test --ignore=cli/ --tb=line -v --junitxml=junit-{envname}.xml {posargs}
 deps =
     {[base]deps}
     mock
-    pytest-cov
     pyfakefs<3.3
     pytest
     salttesting
     configobj
     boto
-    codecov
-    pytest-cov
 
 [testenv:lint]
-basepython = python3
-commands = pylint --rcfile=.pylintrc  srv/
+commands = pylint --rcfile=.pylintrc --jobs=5 srv/ 
```

Comment:
I believe jobs parameter is redundant here since it is already in pylintc

Reply:
I simply pulled this from SES5 in order to get environments working.  If we have been moving parameters from command line to config file, I am not aware.